integrations/googlecalendar/src/actions/sync/event-create.ts

```diff
@@ -0,0 +1,50 @@
+import { getClient } from 'src/client'
+import { mapEvent } from 'src/misc/map-event'
+import { parseError } from 'src/misc/utils'
+import * as bp from '.botpress'
+
+type Merge<T, U> = Omit<T, keyof U> & U
+
+type EventCreateInput = bp.actions.eventCreate.input.Input
+type EventCreateProps = Merge<
+  bp.ActionProps['eventCreate'],
+  {
+    input: Merge<
+      EventCreateInput,
+      {
+        item: Omit<EventCreateInput['item'], 'id'>
+      }
+    >
+  }
+>
+
+export const eventCreate = (async (props: EventCreateProps) => {
+  const { logger, ctx, input } = props
+  try {
+    const { item } = input
+    const { calendar } = await getClient(ctx.configuration)
+    const response = await calendar.events.insert({
+      calendarId: ctx.configuration.calendarId,
+      requestBody: {
+        summary: item.summary,
+        description: item.description,
+        location: item.location,
+        start: {
+          // The replaceAll is used to remove the extra quotes from the input created by the studio
+          dateTime: item.startDateTime?.replaceAll('"', ''),
+        },
+        end: {
+          // The replaceAll is used to remove the extra quotes from the input created by the studio
+          dateTime: item.endDateTime?.replaceAll('"', ''),
```

Comment:
nitpick(non-blocking): Is this still true?

Reply:
I don't know... This code was copied so I didn't want to mess with it too much